### ISSUE TITLE
In the visualizer, all velocities are zero

### ISSUE THREAD

**Issue by julskast** [HaliteChallenge/Halite-II#21]

**Comment by lidavidm** [comment]
This is really a leftover from when we had inertia - either we shouldn't display it anymore, or we should have it display the value from the thrust command for that turn  (if any).

**Comment by Snaar** [comment]
+1 on putting thrust command values there

**Comment by Snaar** [comment]
I can take a look at it once I'm done with cpp client and noone else did it by then.

**Comment by harikmenon** [comment]
Yep, Snaar is correct we need the thrust command. @lidavidm and I had this discussion earlier https://github.com/twosigma/Halite-II/issues/467

**Comment by harikmenon** [comment]
@lidavidm is it easy for us show the move command here?

**Comment by Snaar** [comment]
It requires changing halite itself, but theoretically no big deal. I imagine just no time to do it yet since not very high priority.

**Comment by Snaar** [comment]
Oh, I see, he also thought maybe we can do purely in visualizer by keeping track of move command there.

**Comment by harikmenon** [comment]
Yep, thats my thinking  :)
